card_lookup.py

```diff
@@ -49,7 +49,8 @@ def process_card_lookup(matches):
                     for x in range(0, len(group_words) - 1):
                         non_exact_sql = non_exact_sql + " AND card_name LIKE ?"
 
-                    non_exact_sql = non_exact_sql + " ORDER BY card_set_id DESC" # prioritize newer sets
+                    # prioritize newer sets but don't prioritize tokens
+                    non_exact_sql = non_exact_sql + " ORDER BY CASE when card_set_id = 90000 THEN 2 ELSE 1 END, card_set_id DESC" 
```

Comment:
```suggestion
                    non_exact_sql = non_exact_sql + " ORDER BY CASE when card_set_id > 70000 THEN 2 ELSE 1 END, card_set_id DESC"                  
```
card sets 700xx are filled with prebuilts and collab alt art cards. Probably best to de-prioritize them as well.